Moblin/Various/Model/ModelKick.swift

```diff
@@ -85,6 +94,18 @@ extension Model {
         createKickApi()?.setStreamTitle(title: title, onComplete: onComplete)
     }
 
+    private func fetchKickSubscriberBadges() {
+        guard !stream.kickChannelName.isEmpty else { return }
+
+        getKickChannelInfo(channelName: stream.kickChannelName) { channelInfo in
+            DispatchQueue.main.async {
+                if let channelInfo, let subscriberBadges = channelInfo.subscriber_badges {
+                    self.setKickSubscriberBadges(subscriberBadges)
+                }
+            }
+        }
+    }
```

Comment:
I prefer doing this in the kick pusher class. The model does not have to know about this.